internal/rbd/replicationcontrollerserver.go

```diff
@@ -139,6 +144,8 @@ func validateSchedulingDetails(parameters map[string]string) error {
 	val := parameters[imageMirroringKey]
 
 	switch imageMirroringMode(val) {
+	case imageMirrorModeJournal:
+		return nil
```

Comment:
return error message as scheduling is not applicable for journal mode

Reply:
Do you mean log an INFO msg? we definitely do not want the callee to bail out right?

Reply:
>return status.Error(codes.InvalidArgument, "scheduling is only supported for snapshot mode")

scheduling is not supported for journal mode isn't it? we need to return an error message to the user as it's a user input error.  I think that is already handled below at line 155.we can remove this check

Reply:
Added the comment as discussed.